bin/jsdoc-to-ts.js

```diff
@@ -13,7 +13,7 @@ const jsdocToTs = require('..');
 process.on('uncaughtException', err => console.error(err.stack));
 
 function displayHelp () {
-	let e = 'node ' + path.relative(process.cwd(), __filename);
+	let e = 'node ' + path.relative(process.cwd(), __filename);	// eslint-disable-line no-undef
```

Comment:
Could you share the reason?
When I tested `npm run lint` without eslint-disable-line but there was no error.

Reply:
Yes you're right, I will revert this change